python-rhsm.spec

```diff
@@ -30,8 +33,10 @@ BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 %if 0%{?sles_version}
 Requires: python-m2crypto
 %else
+%if %use_m2crypto
```

Comment:
Shouldn't the sles_version predicate be nested inside the use_m2crypto one?  In other words, shouldn't line 36 here go above line 33.